File: content/37/a.go
package main
func main() {
   n := 1
   // example 1
   s := ""
   if n > 0 {
      s = "+"
   } else if n < 0{
      s = "-"
   } else {
      s = "zero"
   }
   // example 2
   s2 := map[bool]string{true: "+", false: "-"}[n > 0]
   // print
   println(s == "+", s2 == "+")
}

File: content/37/a.js
let n = 1;
// example 1
let s;
if (n > 0) {
   s = '+';
} else if (n < 0) {
   s = '-';
} else {
   s = 'zero';
}
// example 2
let s2 = n > 0 ? '+' : '-';
// print
console.log(s == '+', s2 == '+');
